keyboards/eyeohdesigns/s50/s50.h

```diff
@@ -0,0 +1,238 @@
+         k33, k34,k20, k21, k22, k23, k24, k25, k26, k27, k64, k65, k50, k55, k46,\
+        k35, k36, k37, k66,          k40,                      k42, k43,   k44    \
+) \
+{ \
+   {k00, k01,  k02, k03,  k04,  k05,  k06,  k07}, \
+   {k10, k11,  k12, k13,  k14,  k15,  k16,  k17}, \
+   {k20, k21,  k22, k23,  k24,  k25,  k26,  k27}, \
+   {k30, k31,  k32, k33,  k34,  k35,  k36,  k37}, \
+   {k40, KC_NO,k42, k43,  k44,  KC_NO,k46,  k47}, \
+   {k50, k51,  k52, KC_NO,k54,  k55,  k56,  k57}, \
+   {k60, k61,  k62, k63,  k64,  k65,  k66,  KC_NO}, \
+}
+#define ISO_Split( \
+             k31, k00, k01, k02, k03, k04, k05, k06, k07, k60, k61, k52,     k47, \
+   k30,                                                                  k54,     \
+            k32,  k10, k11, k12, k13, k14, k15, k16, k17, k62, k63, k51,      k56,\
+         k33, k34,k20, k21, k22, k23, k24, k25, k26, k27, k64, k65, k50, k55, k46,\
+         k35, k36, k37,    k66,      k67,      k40,     k41,   k42, k43, k44,  k45\
+) \
+{ \
+   {k00, k01, k02, k03,  k04,  k05, k06, k07}, \
+   {k10, k11, k12, k13,  k14,  k15, k16, k17}, \
+   {k20, k21, k22, k23,  k24,  k25, k26, k27}, \
+   {k30, k31, k32, k33,  k34,  k35, k36, k37}, \
+   {k40, k41, k42, k43,  k44,  k45, k46, k47}, \
+   {k50, k51, k52, KC_NO,k54,  k55, k56, KC_NO}, \
+   {k60, k61, k62, k63,  k64,  k65, k66, k67}, \
+}
+#define ISO_Split_Num( \
+             k31, k00, k01, k02, k03, k04, k05, k06, k07, k60, k61, k52, k57, k47,\
+   k30,                                                                           \
+            k32,  k10, k11, k12, k13, k14, k15, k16, k17, k62, k63, k51, k54, k56,\
+         k33, k34,k20, k21, k22, k23, k24, k25, k26, k27, k64, k65, k50, k55, k46,\
+         k35, k36, k37, k66,      k67,      k40, k41,          k42, k43,     k44  \
+) \
+{ \
+   {k00, k01, k02, k03,  k04, k05,  k06, k07}, \
+   {k10, k11, k12, k13,  k14, k15,  k16, k17}, \
+   {k20, k21, k22, k23,  k24, k25,  k26, k27}, \
+   {k30, k31, k32, k33,  k34, k35,  k36, k37}, \
+   {k40, k41, k42, k43,  k44, KC_NO,k46, k47}, \
+   {k50, k51, k52, KC_NO,k54, k55,  k56, k57}, \
+   {k60, k61, k62, k63,  k64, k65,  k66, k67}, \
+}
+#define tester( \
+             k31, k00, k01, k02, k03, k04, k05, k06, k07, k60, k61, k52, k57, k47,\
+   k30,                                                                           \
+            k32,  k10, k11, k12, k13, k14, k15, k16, k17, k62, k63, k51, k54, k56,\
+         k33, k34,k20, k21, k22, k23, k24, k25, k26, k27, k64, k65, k50, k55, k46,\
+    k35, k36, k37, k66,      k67,      k40, k41,          k42, k43,     k45, k44  \
+) \
+{ \
+   {k00, k01, k02, k03,  k04, k05,  k06, k07}, \
+   {k10, k11, k12, k13,  k14, k15,  k16, k17}, \
+   {k20, k21, k22, k23,  k24, k25,  k26, k27}, \
+   {k30, k31, k32, k33,  k34, k35,  k36, k37}, \
+   {k40, k41, k42, k43,  k44, k45,  k46, k47}, \
+   {k50, k51, k52, KC_NO,k54, k55,  k56, k57}, \
+   {k60, k61, k62, k63,  k64, k65,  k66, k67}, \
+}
+#endif
```

Comment:
```suggestion
```

Reply:
I've made all the other changes, but making this change breaks the code and will not allow me to compile.

Reply:
Got it fixed and committed.